stan/math/prim/scal/prob/chi_square_rng.hpp

```diff
@@ -1,49 +1,52 @@
 #ifndef STAN_MATH_PRIM_SCAL_PROB_CHI_SQUARE_RNG_HPP
 #define STAN_MATH_PRIM_SCAL_PROB_CHI_SQUARE_RNG_HPP
 
-#include <boost/random/chi_squared_distribution.hpp>
-#include <boost/random/variate_generator.hpp>
 #include <stan/math/prim/scal/err/check_consistent_sizes.hpp>
-#include <stan/math/prim/scal/err/check_nonnegative.hpp>
-#include <stan/math/prim/scal/err/check_not_nan.hpp>
+#include <stan/math/prim/scal/err/check_finite.hpp>
 #include <stan/math/prim/scal/err/check_positive_finite.hpp>
-#include <stan/math/prim/scal/fun/constants.hpp>
-#include <stan/math/prim/scal/fun/multiply_log.hpp>
-#include <stan/math/prim/scal/fun/value_of.hpp>
-#include <stan/math/prim/scal/fun/gamma_p.hpp>
-#include <stan/math/prim/scal/fun/digamma.hpp>
-#include <stan/math/prim/scal/fun/grad_reg_inc_gamma.hpp>
-#include <stan/math/prim/scal/meta/include_summand.hpp>
+#include <stan/math/prim/scal/meta/length.hpp>
+#include <stan/math/prim/scal/meta/scalar_seq_view.hpp>
+#include <stan/math/prim/scal/meta/VectorBuilder.hpp>
+#include <boost/random/chi_squared_distribution.hpp>
+#include <boost/random/variate_generator.hpp>
 
 namespace stan {
   namespace math {
-
     /**
-     * Return a pseudorandom chi squared variate with the nu degrees of freedom
-     * using the specified random number generator.
+     * Return a pseudorandom chi squared variate with the nu degrees of
+     * freedom using the specified random number generator.
      *
+     * nu can be a scalar, a std::vector, an Eigen::Vector, or an
+     * Eigen::RowVector.
```

Comment:
There is no `Eigen::Vector` or `Eigen::RowVector`---it's `Eigen::Matrix<T, -1, 1>` or `Eigen::Matrix<T, 1, -1>`. 

The comments should be either real code with code escapes or should be just prose.  So this can be just "standard vector or Eigen vector or row vector".

Either use code and quote it as code, or just talk about it.  Here, I'd recommend